data/language/en-GB.txt

```diff
@@ -2531,8 +2531,8 @@ STR_2523    :Show map
 STR_2524    :Screenshot
 ### The following need to be reordered to match SDL_keycode layout.
 STR_2525    :???
-STR_2526    :???
-STR_2527    :???
+STR_2526    :Show 'real' names of staff
+STR_2527    :{SMALLFONT}{BLACK}Toggle between showing 'real' names of staff and staff numbers
```

Comment:
These two strings should be in the bottom following the number order, so it should be STR_6130 and STR_6131 :)